seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4180,6 +4180,9 @@
         "Here's a sample object:",
         "<blockquote>var cat = {<br>  \"name\": \"Whiskers\",<br>  \"legs\": 4,<br>  \"tails\": 1,<br>  \"enemies\": [\"Water\", \"Dogs\"]<br>};</blockquote>",
         "Objects are useful for storing data in a structured way, and can represent real world objects, like a cat.",
+        "In this example, all the properties are stored as strings, such as - <code>\"name\"</code>, <code>\"legs\"</code>, and <code>\"tails\"</code>. However, you can also use numbers as properties. You can even omit the quotes for single-word string properties, as follows:",
+        "<blockquote>var anotherObject = {<br>  make: \"Ford\",<br>  5: \"five\",<br>  \"model\": \"focus\"<br>};</blockquote>",
+        "However, if your object has any non-string properties, JavaScript will automatically typecast them to their corresponding string types.",
```

Comment:
I would change `"...JavaScript will automatically typecast them to their corresponding string types."`

to:

`"JavaScript will automatically typecast them as strings."`